bentoml/frameworks/onnx.py

```diff
@@ -138,15 +138,17 @@ def load(self, path):
     def set_dependencies(self, env: BentoServiceEnv):
         if self.backend == "onnxruntime":
             env.add_pip_packages(["onnxruntime"])
+        elif self.backend == "onnxruntime-gpu":
+            env.add_pip_packages(["onnxruntime-gpu"])
```

Comment:
can we add a check in the `__init__` method that the package for the specified backend can be imported without error?

Reply:
I think this is a good suggestion.  And I can see the backend package such as `onnxruntime` is not available during the time users are packing artifacts.

Reply:
When packing artifacts, the backend package must be available